src/Directory.Build.props

```diff
@@ -15,7 +15,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Neo" Version="3.0.0-CI01116" />
+    <PackageReference Include="Neo" Version="3.0.0-CI01119" />
```

Comment:
@shargon  We need update `NativeContract` name and make compatible with `Net5.0` to use `3.0.0-CI01119`. I think we should commit in another pr. Please `3.0.0-CI01116` in this pr.

Reply:
Updated.